diesel/src/query_dsl/load_dsl.rs

```diff
@@ -9,39 +9,31 @@ use types::HasSqlType;
 /// Methods to execute a query given a connection. These are automatically implemented for the
 /// various query types.
 pub trait LoadDsl<Conn: Connection>: AsQuery + Sized where
-    Self::Query: QueryFragment<Conn::Backend> + QueryId,
     Conn::Backend: HasSqlType<Self::SqlType>,
 {
     /// Executes the given query, returning an `Iterator` over the returned
     /// rows.
     fn load<'a, U>(self, conn: &Conn) -> QueryResult<Vec<U>> where
-        U: Queryable<Self::SqlType, Conn::Backend> + 'a,
-    {
-        conn.query_all(self)
-    }
+        U: Queryable<Self::SqlType, Conn::Backend> + 'a;
```

Comment:
This git churn irks me to no end, but there is no way to write this in a way that doesn't churn. I would have added a random semicolon on a separate line if it would have let me keep this comma. YES I AM THAT DRUNK WITH STYLE GUIDE POWER